### ISSUE TITLE
AAB option on GUI publishing menu

### ISSUE THREAD

**Issue by CCC200** [airsdk/Adobe-Runtime-Support#879]
## Feature Request
The ability to select an AAB compiler option in the GUI menu used in Animate would be very useful, as google is requiring all new apps to use this format after August. An extra bullet on the processor list would suffice, or a toggle between APK/AAB.

![AIR Menu](https://i.imgur.com/SHqqmdy.png)

## Known Workarounds
Manually compiling SWFs and using the command line to create an AAB is possible, but complicates the workflow compared to a built-in solution.

**Comment by ajwfrost** [comment]
Yes good point, I have just emailed the Animate team to see if they can update this.
As a short-term option we may be able to use the ADT config file, similar to how we worked around things prior to them adding the "ARMv8" option (which may be needed anyway, in case folk are still using other IDEs that don't have the ability to add this..)

thanks
